english/sleep/sit-down-to-meditate/sit-down-to-meditate.md

```diff
@@ -0,0 +1 @@
+I sit down and light an incense stick. Close my eyes and try to focus on my thoughts, try to feel relaxed and light as air. I meditate on the smell and color of the marshmallow walls, as they transform slowly into talking gurus.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.